### ISSUE TITLE
Really interested

### ISSUE THREAD

**Issue by MrMoronIV** [flutter-webrtc/flutter-mediasoup-client#1]
I'm really interested to see this project happening. I'm running a nodejs mediasoup server and would really like to use it with a flutter app!

**Comment by cloudwebrtc** [comment]
Yeah, I am working hard to develop it. When I finish it, using flutter for ```multiparty video chat``` / ```live broadcast``` will be very easy.

**Comment by MrMoronIV** [comment]
amazing! Thank you so much!

**Comment by passsionforprogramming** [comment]
This looks pretty amazing. Is it done?

**Comment by cloudwebrtc** [comment]
Can continue now.
https://github.com/flutter-webrtc/flutter-webrtc/commit/11c1fd1137be6333c2252a20e999b2d6b53632db

**Comment by olzii** [comment]
> Can continue now.
> [flutter-webrtc/flutter-webrtc@11c1fd1](https://github.com/flutter-webrtc/flutter-webrtc/commit/11c1fd1137be6333c2252a20e999b2d6b53632db)

Is it possible to use plugin now ? Are you resuming it ?

**Comment by ajw227** [comment]
> Can continue now.
> [flutter-webrtc/flutter-webrtc@11c1fd1](https://github.com/flutter-webrtc/flutter-webrtc/commit/11c1fd1137be6333c2252a20e999b2d6b53632db)

Hello. I'm interesting this project.
@cloudwebrtc  Do you have a plan for develop this project?

**Comment by Blancduman** [comment]
Currently, I working on porting. Need to finish native implementation for handler, device and transport. Other things must be done, but still untested [hehe]

**Comment by borisgrigorov** [comment]
Is anyone working on this?

**Comment by amenabe22** [comment]
Is there any other well developed library out there ?
